### DISCUSSION TITLE
How to change domain name of a site

### DISCUSSION THREAD

**Discussion by gstazure** [EasyEngine/easyengine#1539] [Support Requests]
Hi, 

I have a development site _mysitedev.com_ working on easyengine. Everything is good.
Now I want to point a real domain say _mysitelive.com_  to it. 
Someone please guide, if there are a set of commands which can accomplish this. Will avoid the hassle of migrating a WP site.

Thanks a ton.
Prakash

**Comment by DavidCamejo** [comment]
You can use WPvivid Backup Plugin. With it you can make a backup and then restore it on a clean WordPress installation with any domain easily. I tried with several plugins and WPvivid wins by a lot. Trust me. Greetings!

**Comment by tosinamuda** [comment]
anyway to do this in 2022?

**Comment by marcoandrei** [comment]
@gstazure and @tosinamuda 

There is no easy way to do that. You need to create a new site and migrate your current site to it. Look at https://github.com/EasyEngine/easyengine/discussions/1770.

Saúde!
